<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget_center" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="Layout_System"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_right" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/basic icons/resources/AnalyzerLogo300.jpg</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <widget class="QLCDNumber" name="LCD_DVFS_MAX">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>200</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_DVFS_MIN">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>200</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>162</x>
            <y>183</y>
            <width>91</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>DVFS Min</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_12">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>184</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>DVFS Max</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_BUS">
          <property name="geometry">
           <rect>
            <x>151</x>
            <y>476</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_DDR">
          <property name="geometry">
           <rect>
            <x>3</x>
            <y>476</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>164</x>
            <y>113</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>GPU Freq</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_GPU_FREQ">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>130</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_GPU_UTIL">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>130</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>459</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>DDR</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>460</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>BUS</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>113</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>GPU Util</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_GPU">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>408</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU_G0">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>338</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>157</x>
            <y>253</y>
            <width>66</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>PST</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU_G1">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>338</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_PST">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>270</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>388</y>
            <width>66</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>GPU</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>254</y>
            <width>66</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>APT</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>322</y>
            <width>66</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>CPU0</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_APT">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>270</y>
            <width>141</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <stylestrategy>PreferDefault</stylestrategy>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::Panel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>321</y>
            <width>66</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>CPU1</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU0">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>21</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="smallDecimalPoint">
           <bool>false</bool>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU1">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>21</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU2">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>21</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU3">
          <property name="geometry">
           <rect>
            <x>227</x>
            <y>21</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>73</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU6">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>73</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU7">
          <property name="geometry">
           <rect>
            <x>226</x>
            <y>73</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_CPU5">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>73</y>
            <width>71</width>
            <height>31</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="numDigits">
           <number>4</number>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_16">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>4</y>
            <width>101</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Little Cores</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>56</y>
            <width>71</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Big Cores</string>
          </property>
         </widget>
         <widget class="QSlider" name="SlideBar">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>530</y>
            <width>291</width>
            <height>29</height>
           </rect>
          </property>
          <property name="minimum">
           <number>13</number>
          </property>
          <property name="maximum">
           <number>35</number>
          </property>
          <property name="value">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_button" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_Start">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>24</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Stop">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>26</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionInstall2Device"/>
   <addaction name="actionEventRecord"/>
   <addaction name="actionEventStop"/>
   <addaction name="actionEventReplay"/>
   <addaction name="actionCaptureScreen"/>
   <addaction name="actionMP4_Recording"/>
   <addaction name="actionPerformanceMode"/>
   <addaction name="actionNormalMode"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/plug42.png</normaloff>:/basic icons/resources/icons/plug42.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to device</string>
   </property>
  </action>
  <action name="actionInstall2Device">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/smartphone34.png</normaloff>:/basic icons/resources/icons/smartphone34.png</iconset>
   </property>
   <property name="text">
    <string>Install2Device</string>
   </property>
   <property name="toolTip">
    <string>Install showfreq to device</string>
   </property>
  </action>
  <action name="actionEventRecord">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/start.png</normaloff>:/basic icons/resources/start.png</iconset>
   </property>
   <property name="text">
    <string>EventRecord</string>
   </property>
   <property name="toolTip">
    <string>Recording event</string>
   </property>
  </action>
  <action name="actionEventStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/stop.png</normaloff>:/basic icons/resources/stop.png</iconset>
   </property>
   <property name="text">
    <string>EventStop</string>
   </property>
   <property name="toolTip">
    <string>Stop recording event</string>
   </property>
  </action>
  <action name="actionEventReplay">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/document65.png</normaloff>:/basic icons/resources/icons/document65.png</iconset>
   </property>
   <property name="text">
    <string>EventReply</string>
   </property>
   <property name="toolTip">
    <string>Replay event</string>
   </property>
  </action>
  <action name="actionCaptureScreen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/floppy13.png</normaloff>:/basic icons/resources/icons/floppy13.png</iconset>
   </property>
   <property name="text">
    <string>CaptureScreen</string>
   </property>
   <property name="toolTip">
    <string>Screen capture</string>
   </property>
  </action>
  <action name="actionMP4_Recording">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/folder81.png</normaloff>:/basic icons/resources/icons/folder81.png</iconset>
   </property>
   <property name="text">
    <string>MP4 Recording</string>
   </property>
   <property name="toolTip">
    <string>Recording MP4 movie file</string>
   </property>
  </action>
  <action name="actionPerformanceMode">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/power button2.png</normaloff>:/basic icons/resources/icons/power button2.png</iconset>
   </property>
   <property name="text">
    <string>PerformanceMode</string>
   </property>
   <property name="toolTip">
    <string>Performance Mode</string>
   </property>
  </action>
  <action name="actionNormalMode">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/basic icons/resources/icons/power107.png</normaloff>:/basic icons/resources/icons/power107.png</iconset>
   </property>
   <property name="text">
    <string>NormalMode</string>
   </property>
   <property name="toolTip">
    <string>Normal Mode</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
